Exiled.API/Features/Player.cs

```diff
@@ -594,35 +594,35 @@ public bool NoClipEnabled
         /// <summary>
         /// Gets a value indicating whether or not the player is dead.
         /// </summary>
-        public bool IsDead => Role?.Team == Team.RIP;
+        public bool IsDead => Role?.Team is Team.RIP;
 
         /// <summary>
         /// Gets a value indicating whether or not the player's <see cref="RoleType"/> is any NTF rank.
         /// Equivalent to checking the player's <see cref="Team"/>.
         /// </summary>
-        public bool IsNTF => Role?.Team == Team.MTF;
+        public bool IsNTF => Role?.Team is Team.MTF;
 
         /// <summary>
         /// Gets a value indicating whether or not the player's <see cref="RoleType"/> is any Chaos rank.
         /// Equivalent to checking the player's <see cref="Team"/>.
         /// </summary>
-        public bool IsCHI => Role?.Team == Team.CHI;
+        public bool IsCHI => Role?.Team is Team.CHI;
 
         /// <summary>
         /// Gets a value indicating whether or not the player's <see cref="RoleType"/> is any SCP.
         /// Equivalent to checking the player's <see cref="Team"/>.
         /// </summary>
-        public bool IsScp => Role?.Team == Team.SCP;
+        public bool IsScp => Role?.Team is Team.SCP;
 
         /// <summary>
         /// Gets a value indicating whether or not the player's <see cref="RoleType"/> is any human rank.
         /// </summary>
-        public bool IsHuman => Role is not null && Role.Is(out HumanRole _);
+        public bool IsHuman => Role?.Team is not Team.SCP and not Team.RIP;
```

Comment:
No

Reply:
Why, type-cast is faster that 2 get-methods?

Reply:
` Gets a value indicating whether or not the player's <see cref="RoleType"/> is any human rank.`
Human rank is defined by `HumanRole`, and even if it's faster than two get methods, we can't just assume that NW won't add more non-human roles. "We can add a `and not Team.NewNonHumanTeam` if that's the case", yes, but it goes against the maintainability and it leads to more issues.

Reply:
Okay, i'll revert it